DataFormats/BTauReco/interface/TemplatedSoftLeptonTagInfo.h

```diff
@@ -163,8 +163,8 @@ TaggingVariableList TemplatedSoftLeptonTagInfo<REF>::taggingVariables(void) cons
     const SoftLeptonProperties & data = m_leptons[i].second;
     list.insert( TaggingVariable(btau::leptonQuality,  data.quality(SoftLeptonProperties::Quality::leptonId, false)), true );
     list.insert( TaggingVariable(btau::leptonQuality2, data.quality(SoftLeptonProperties::Quality::btagLeptonCands, false)), true );
-    list.insert( TaggingVariable(btau::trackSip2dSig,  data.sip2d),    true );
-    list.insert( TaggingVariable(btau::trackSip3dSig,  data.sip3d),    true );
+    list.insert( TaggingVariable(btau::trackSip2dSig,  data.sip2d),    true ); 
+    list.insert( TaggingVariable(btau::trackSip3dSig,  data.sip3d),    true ); 
```

Comment:
Can you remove spaces or tabs at the end of these two lines? Otherwise, this will probably create unnecessary merge conflicts with/for #11247, depending on which PR gets merged first.